packages/ssi-service/src/feedback-panel/feedback.js

```diff
@@ -0,0 +1,293 @@
+import html from 'html-template-tag';
+import { GraphQLClient } from 'graphql-request';
+import styles from './feedback.css';
+
+window.customElements.define('op-feedback', class extends HTMLElement {
+  constructor () {
+    super();
+
+    this.graphqlClient = new GraphQLClient('https://localhost:8080/graphql', {
+      headers: {
+        Authorization: 'Basic b25lX3BvcnRhbF9hcGk6PUw5dm5CK2RxNVoqcVo5cA=='
```

Comment:
Credentials are Public. please use the environment variables also remove this from your commit history.

Reply:
@riginoommen This is just a Token. This shouldn't be much of a problem as long as the Authorisation backend has proper checks for it.
Also, this is not exactly a secret, as the client end js is a white box.

A solution to make this secure is to create a unique token for this purpose, and add limited access for it, to just submit a feedback/bug report.

Reply:
Even if its just a token still it should NEVER be pushed to a public repo. Use ENV variables or any other appropriate approach. It's not just about security but also about practices.

Reply:
Removed it and added it in a `secrets.json` file